tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -377,17 +376,17 @@ where
 				)?;
 				calls.push(OpaqueCall::from_tuple(&(
 					node_metadata_repo.get_from_metadata(|m| m.unshield_funds_call_indexes())??,
+					shard,
 					beneficiary,
 					value,
-					shard,
```

Comment:
I don't see a matching change in the pallet though

Reply:
Thanks, you are right. The change is introduced from upstream. Revert it back.